api/python/ai/chronon/repo/hub_utils.py

```
@@ -182,7 +139,7 @@ def deploy(self, date=None, force_recompute=False, plan=False):
     Raises:
         ValueError: If the object cannot be compiled or uploaded
     """
-    _compile_and_upload_to_branch(self)
+    upload_to_branch(self)
```

Pylint (3.3.7)

[error] No value for argument 'zipline_hub' in function call

(E1120)